client/web/webpack.config.js

```diff
@@ -232,6 +232,10 @@ const config = {
       // https://github.com/joshwnj/react-visibility-sensor/issues/148
       'react-visibility-sensor': path.resolve(ROOT_PATH, 'node_modules/react-visibility-sensor/visibility-sensor.js'),
     },
+    fallback: {
+      path: require.resolve('path-browserify'),
+      util: require.resolve('util/'),
+    },
```

Comment:
Still not clear to be why I had to add these for pnpm.

Reply:
What does "fallback" do?

Reply:
Something to do with redirecting imports if it's not found, which is useful for browserified node libraries (I think).